src/infra/repos/postgresql/tag-repository.ts

```diff
@@ -24,4 +24,22 @@ export class TagRepository implements AddTagRepository, LoadTagByIdRepository {
     })
     return tag
   }
+
+  async update (params: UpdateTagRepository.Params): Promise<UpdateTagRepository.Result> {
+    const { name, id } = params
+    const tag = await db.tag.update({
+      data: {
+        name
+      },
+      where: {
+        id
+      },
+      select: {
+        id: true,
+        name: true,
+        color: true
+      }
+    })
```

Comment:
pode remover o select já que todos os campos estão sendo selecionados
```suggestion
    const tag = await db.tag.update({
      data: {
        name
      },
      where: {
        id
      }
    })
```